articles/portal/ptl-how-restrict-access-vcloud-api.md

```diff
@@ -1,36 +1,37 @@
 ---
-title: How to restrict access to the vCloud API
-description: Shows how to create a whitelist of IP addresses to restrict access to the vCloud API
+title: How to restrict access to VMware Cloud Director/vCloud Director
+description: Shows how to create an allowlist of IP addresses to restrict access to the vCloud API and VMware Cloud Director/vCloud Director
 services: portal
-author: Sue Highmoor
-reviewer:
-lastreviewed: 15/11/2019
+author: shighmoor
+reviewer: shighmoor
+lastreviewed: 01/12/2020
 
 toc_rootlink: How To
 toc_sub1: 
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: Restrict access to the vCloud API
+toc_title: Restrict access to vCloud Director
```

Comment:
I'm not familiar enough to know whether the toc_title should exactly match other text in document. I don't see an exact match shown here in github.
Should this title also have been updated to say 'VMware Cloud Director/vCloud Director' as it has been updated in other locations?

Reply:
It doesn't have to match exactly, because we have less space for the toc title  than for the main title - but you're right, it should be VMware Cloud Director/vCloud Director. I'll fix that.